server/api/topics.js

```diff
@@ -58,4 +58,15 @@ router.get("/:topic_id/questions", (req, res) => {
 			res.json(500);
 		});
 });
+router.post("/:topic_id/createQuestion", (req, res) => {
+	const newQuestion = req.body.question_text;
+	const id = req.params.topic_id;
+	topicDb
+		.createNewQuestion(id, newQuestion)
+		.then(() => res.send("Question created!"))
+		.catch((err) => {
+			console.error(err);
+			res.json(500);
+		});
```

Comment:
So since I can see that you are fast enough to create endpoints. What about *error handling*?
In other words, what happens when we get a request without a `question_text` in the body?

Reply:
Lemme explain what it should happen and we can discuss how to implement it. It should return a [`400 Bad request`](https://httpstatuses.com/400) status code with a message for the user like _Bad request. The `question_text` is required_

Reply:
Done! 
also added error handling for creating a topic